src/pages/community.html

```diff
@@ -1,9 +1,37 @@
 {{> header}}
 
 <div class="container">
+  <div class="fixed-links">
+    <div class="dropdown" >
+      <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">Community<span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li><a href="#mailing_lists">Mailing Lists</a></li>
+        <li><a href="#meetup_groups">Meetup Groups</a></li>
+        <li><a href="#people">People</a></li>
+        <li><a href="#powered_by_apex">Powered By Apex</a></li>   
+      </ul>
+    </div>
+
+    <div class="dropdown">
+      <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">Contributing<span class="caret"></span></button>
+      <ul class="dropdown-menu">
+        <li><a href="#issue_tracking">Issue Tracking</a></li>
+        <li><a href="#release_process">Release Process</a></li>
+        <li><a href="#security_process">Security Process</a></li>
```

Comment:
These links don't make sense in the menu. I would suggest to remove this dropdown and instead add a contributing link under community that links to the "Contributing" section of the page. There would be larger content changes needed to justify a separate top level dropdown.